### DISCUSSION TITLE
No elevation in Bulldozer? Not able to edit Keypoints or roads?

### DISCUSSION THREAD

**Discussion by TheRealBJTS** [jetelain/ArmaRealMap#171] [Q&A] [answered]
Hey there,

I've seen to run into some trouble with exporting my .grma3m file into Terrain Builder Files. I have followed the steps of the README and imported the heightmap, sat image, and mask image. I can see the elevation change when moving the cursor over the heightmap in Terrain Builder, however when I connect everything to Bulldozer there is no elevation at all, it is all set to 0. I am not sure what exactly is the issue here and would appreciate any insight.

I also cannot seem to find to edit any roads or keypoints on the map, and would like some assistance in that as well. I don't have much experience in Terrain Builder so any help would be appreciated!

Thanks!

**Comment by jetelain** [comment, marked as answer]
Hi,

You have to use the "Rebuild terrain" button to get the height map loaded. It's located in mapframe properties, in the the processing tab.

**Reply by TheRealBJTS** [reply]
Thanks! I forgot that I should probably rebuild the terrain after importing the heightmap.

Any insight on how to edit the road layers and keypoints in Terrain Builder?
